bpf/bpf_host.c

```diff
@@ -495,7 +494,7 @@ handle_to_netdev_ipv6(struct __ctx_buff *ctx, struct trace_ctx *trace, __s8 *ext
 		return hdrlen;
 
 	if (likely(nexthdr == IPPROTO_ICMPV6)) {
-		ret = icmp6_host_handle(ctx, ETH_HLEN + hdrlen);
+		ret = icmp6_host_handle(ctx, ETH_HLEN + hdrlen, ext_err);
```

Comment:
This user would now also pick up the changed behaviour in `icmp6_host_handle()`. Is that intentional, or should we pass a flag that we don't want the NS responder to be active for this code path?